src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    readonly color: string,
+    readonly a: number,
+    readonly b: number,
+    readonly c: number,
+    readonly shape: string = 'triangle',
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('All values must be greater than 0.');
+    }
+
+    const largestSide = Math.max(a, b, c);
+    const sumOfOtherSides = a + b + c - largestSide;
+
+    if (largestSide >= sumOfOtherSides) {
+      throw new Error(
+        'The largest side must not be >= to the sum of the other two sides.',
+      );
+    }
+  }
+
+  getArea(): number {
+    const semiperimeter = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(
+      semiperimeter *
+        (semiperimeter - this.a) *
+        (semiperimeter - this.b) *
+        (semiperimeter - this.c),
+    );
+
+    return Math.floor(area * 100) / 100;
```

Comment:
Would be good to create separate function for rounding(you use `Math.floor(...* 100) / 100` at least 3 times
Remember about DRY rule